packages/reports/tests/acceptance/multi-datasource-test.js

```diff
@@ -190,8 +190,8 @@ module('Acceptance | multi-datasource report builder', function (hooks) {
       .dom('.export__download-link')
       .hasAttribute(
         'href',
-        'https://data2.naviapp.io/v1/data/inventory/day/container;show=desc/displayCurrency;show=desc/?dateTime=P3D%2Fcurrent&metrics=usedAmount%2Crevenue(currency%3DGIL)&filters=container%7Cid-in%5B%222%22%5D&having=usedAmount-gt%5B50%5D&format=csv',
-        'uses csv export from right datasource'
+        /(?=.*?data2.naviapp.io)(?=.*format=csv)(?=.*dateTime=)(?=.*csv&filename=Report%20for%20different%20datasource-\d\d\d\d\d\d\d*T\d\d\d\d\d\d)/,
```

Comment:
can we still test that the entire url is correct regardless of the filename (`https://data2.naviapp.io/v1/data/inventory/day/container;show=desc/displayCurrency;show=desc/?dateTime=P3D%2Fcurrent&metrics=usedAmount%2Crevenue(currency%3DGIL)&filters=container%7Cid-in%5B%222%22%5D&having=usedAmount-gt%5B50%5D&format=csv`) and just add the filename?

Reply:
Since we are having the date format I have added regex